### PR TITLE
Use FormUrlEncodedContent to build query

### PR DESCRIPTION
**Pull request by scythx**

### CODE REVIEW

File: LoklakDotNet/Loklak.cs [hunk 1]
```diff
@@ -104,20 +110,23 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
...
-            var qs = "q=" + q.getQueryString() + "&count=" + count.ToString() + "&source=" + source;
+            Dictionary<string,string> qs = new Dictionary<string,string>();
+
+            qs.Add("q",q.getQueryString());
+            qs.Add("count",count.ToString());
+            qs.Add("source",source);
             if(fields!= null)
             {
-                qs += "&fields=" + string.Join(",", fields);
+                qs.Add("fields",string.Join(",",fields));
```
Comment (sudheesh001) [inline LoklakDotNet/Loklak.cs]: please leave a space after the `,` making it `, fields` instead of `,fields`

File: LoklakDotNet/Loklak.cs [hunk 2]
```diff
@@ -104,20 +110,23 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
...
             }
             if(limit!=-1)
             {
-                qs += "&limit=" + limit;
+                qs.Add("limit",limit.ToString());
```
Comment (sudheesh001) [inline LoklakDotNet/Loklak.cs]: Same here, please leave a space after the `,` making it as `, limit`

File: LoklakDotNet/Loklak.cs [hunk 3]
```diff
@@ -104,20 +110,23 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
...
             }
             if(timeZoneOffset!=-1)
             {
-                qs += "&timeZoneOffset=" + timeZoneOffset;
+                qs.Add("timeZoneOffset",timeZoneOffset.ToString());
```
Comment (sudheesh001) [inline LoklakDotNet/Loklak.cs]: Similar here too, `, timeZoneOffset...`

File: LoklakDotNet/Loklak.cs [hunk 4]
```diff
@@ -132,16 +141,19 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
...
-            var qs = "text=" + text + "padding=" + padding.ToString() + "uppercase=" + uppercase;
+            Dictionary<string,string> qs = new Dictionary<string,string>();
+
+            qs.Add("text",text);
+            qs.Add("padding",padding.ToString());
+            qs.Add("uppercase",uppercase ? "1" : "0");
             if (!string.IsNullOrWhiteSpace(color_text))
             {
-                qs += "&color_text=" + color_text;
+                qs.Add("color_text",color_text);
```
Comment (sudheesh001) [inline LoklakDotNet/Loklak.cs]: Same here `, color_text`

File: LoklakDotNet/Loklak.cs [hunk 5]
```diff
@@ -132,16 +141,19 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
...
             }
             if (!string.IsNullOrWhiteSpace(color_background))
             {
-                qs += "&color_background=" + color_background;
+                qs.Add("color_background",color_background);
```
Comment (sudheesh001) [inline LoklakDotNet/Loklak.cs]: `, color_background`

File: LoklakDotNet/Loklak.cs [hunk 6]
```diff
@@ -87,8 +90,11 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
         /// <returns></returns>
         public async Task<string> user(string screen_name, int follower_count=0, int following_count=0)
         {
-            var qs = "screen_name=" + screen_name + "&followers=" + follower_count.ToString() + "&following=" + following_count.ToString();
-            System.Diagnostics.Debug.WriteLine(qs);
+            Dictionary<string,string> qs = new Dictionary<string,string>();
+
+            qs.Add("screen_name",screen_name);
+            qs.Add("followers",follower_count.ToString());
```
Comment (sudheesh001) [inline LoklakDotNet/Loklak.cs]: Change this to be `, follower_count. ...` need the spacing after the `,` so that it looks more pleasing to read this.

**Review by sudheesh001 — CHANGES_REQUESTED** [review]

**Comment by scythx** [comment]
@sudheesh001 Updated.

**Comment by jayvdb** [comment]
@sudheesh001 ping.

**Review by sudheesh001 — APPROVED** [review]

**Pull request merged**